### PR TITLE
Add test languages for entities

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Related to https://github.com/jhipster/generator-jhipster/pull/6164

By adding these tests, I saw 2 missings `_entity_xxx.json`

_____

- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "pretest": "eslint .",
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
-    "test": "mocha --timeout 20000 --slow 0 --reporter spec test/*.spec.js"
+    "test": "mocha --timeout 20000 --slow 0 --reporter spec"
```
Comment (deepu105) [inline package.json]: please look at my comment on the original commit. This should work as it is. This helps to exclude files that are actually not tests. Like the utils

File: test/templates/all-languages/.yo-rc.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "clientPackageManager": "yarn",
+    "applicationType": "monolith",
+    "testFrameworks": [
+      "gatling",
+      "protractor"
+    ],
+    "jhiPrefix": "jhi",
+    "enableTranslation": true,
+    "nativeLanguage": "en",
+    "languages": [
```
Comment (deepu105) [inline test/templates/all-languages/.yo-rc.json]: may be we need to add to docs asking people to add language here and in the content file when submitting new languages.

File: package.json (hunk 1 above)
Comment (pascalgrimaud) [inline package.json]: Need to keep this, and modify the package.json with:
```
"test:unit": "mocha --timeout 20000 --slow 0 --reporter spec"
"test": "npm run test:unit -- test/*.spec.js"
```

It's for me ;) Thanks @deepu105

File: test/templates/all-languages/.yo-rc.json (hunk 2 above)
Comment (pascalgrimaud) [inline test/templates/all-languages/.yo-rc.json]: Yes ok, will do it tonight

Comment (pascalgrimaud) [inline test/templates/all-languages/.yo-rc.json]: done here https://github.com/jhipster/jhipster.github.io/pull/465

**Pull request merged**